bundler/spec/support/helpers.rb

```diff
@@ -235,10 +235,21 @@ def global_config(config = nil)
     end
 
     def create_file(path, contents = "")
+      contents = strip_whitespace(contents)
       path = Pathname.new(path).expand_path(bundled_app) unless path.is_a?(Pathname)
       path.dirname.mkpath
-      File.open(path.to_s, "w") do |f|
-        f.puts strip_whitespace(contents)
+      path.write(contents)
+
+      # if the file is a script, create respective bat file on Windows
+      if contents.start_with?("#!")
+        path.chmod(0o755)
+        if Gem.win_platform?
+          bat_path = path.dirname.join(path.basename.to_s + ".bat")
+          bat_path.write(strip_whitespace(<<-SCRIPT))
```

Comment:
This I believe can be simplified to

```suggestion
          path.sub_ext(".bat").write <<~SCRIPT
```